docs/timezone.md

```diff
@@ -211,6 +211,21 @@ For usage examples and a more complete explanation of how this disambiguation wo
 
 If the result is outside the range that `Temporal.Absolute` can represent, then a `RangeError` will be thrown, no matter the value of `disambiguation`.
 
+### timeZone.**getPossibleAbsolutesFor**(_dateTime_: Temporal.DateTime) : array&lt;Temporal.Absolute&gt;
+
+**Parameters:**
+- `dateTime` (`Temporal.DateTime`): A calendar date and wall-clock time to convert.
+
+**Returns:** An array of 0, 1, or 2 `Temporal.Absolute` objects.
```

Comment:
Is it guaranteed to be 0, 1, or 2?

Reply:
@Ms2ger asked the a similar question in https://github.com/tc39/proposal-temporal/pull/498#discussion_r406020942 and I'm not sure we really resolved at the time whether to enforce that or not.

I doubt there is a triply-occurring wall-clock time in any of the supported IANA time zones. You could implement a custom time zone that had one, but you wouldn't be able to use `DateTime.inTimeZone()` to access all three Absolutes anyway, so you'd have to have a function like in the cookbook example to get your own `earlier`/`middle`/`later` disambiguation behaviour.

Reply:
What's the *benefit* of not allowing more than 2?  What if WackoTimeZone decided that instead of repeating the hour one time at a DST transition, it would repeat the same minute 60 times in a row?  Or what if it decided to make the clocks stand still for the whole hour, leading to many thousands of overlaps (one per nanosecond in an hour)?

If there's a benefit to capping this, fine.  I agree that it's not common.  But if there isn't a good reason, I don't see why to impose that limitation.

Reply:
This code doesn't currently impose any limitation, so to make sure I understand, the only change you are suggesting is in the wording of the docs? (I guess I should change the cookbook example as well to use `possible[possible.length - 1]` instead of `possible[1]` as well)